public/src/account/sign-in-view.html

```diff
@@ -139,7 +140,7 @@ <h1>Sign In</h1>
               <button class="link" on-click="_onRegister">I haven't registered yet</button>
             </div>
             <div style="margin-top: 7px;">
-              <button class="link" on-click="_onRequestRecovery">I forgot my handle or password</button>
+              <button class="link" on-click="_onRequestRecovery">I forgot my handle/email or password</button>
             </div>
```

Comment:
Not sure you want to say "i forgot my email" because that's what you need to reset password.

Reply:
Good one.